src/Raven.Client/Documents/Operations/Attachments/GetAttachmentOperation.cs

```diff
@@ -30,22 +35,26 @@ public sealed class GetAttachmentOperation : IOperation<AttachmentResult>
         /// <param name="name">The name of the attachment to be retrieved.</param>
         /// <param name="type">The type of the attachment.</param>
         /// <param name="changeVector">change vector for optimistic concurrency control. If no concurrency control require this should be set to null</param>
+        /// <param name="from">the position at which to start sending data</param>
+        /// <param name="to">the position at which to stop sending data</param>
         /// <remarks>
         /// This constructor sets up an operation to retrieve a specific attachment.
         /// If the <paramref name="changeVector"/> is provided, it ensures that the operation 
         /// corresponds to the specified version of the document.
         /// </remarks>
-        public GetAttachmentOperation(string documentId, string name, AttachmentType type, string changeVector)
+        public GetAttachmentOperation(string documentId, string name, AttachmentType type, string changeVector, long? from = null, long? to = null)
```

Comment:
can we move that to 2nd ctor where `from` and `to` are not null by default? Would like to keep original ctor for binary compatibility